gradle/dependencies.gradle

```diff
@@ -61,7 +61,7 @@ versions += [
   // when updating checkstyle, check whether the exclusion of
```

Comment:
Could you please remove those comments? they are out-of-date after you upgrade the checkstyle version